docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -159,6 +164,23 @@ Additionally, the listener for `pbPreviewReloaded` has been removed.
 |`fbFormBuilderLoaded`|`ez-form-builder-loaded`|
 |`fbFormBuilderUnloaded`|`ez-form-builder-unloaded`
 
+#### Code cleanup
+
+Selected deprecated functions have been dropped.
+
+|Dropped code|Use instead|
+|------------|-----------|
+|`forcedLanguage`|`allowedLanguages`|
+|`formatPHP`|`formatICU`|
+|`formatDate`|`formatFullDateTime`|
```

Comment:
We should specify that these (`formatDate`, `formatShortDate`, etc...) functions belong to: `window.eZ.helpers.timezone` and `formatPHP` belongs to `window.moment.fn`.